sympy/simplify/trigsimp.py

```diff
@@ -427,23 +427,32 @@ def analyse_gens(gens, hints):
 
 
 def trigsimp(expr, **opts):
-    """
-    reduces expression by using known trig identities
-
-    Explanation
-    ===========
-
-    method:
-    - Determine the method to use. Valid choices are 'matching' (default),
-    'groebner', 'combined', and 'fu'. If 'matching', simplify the
-    expression recursively by targeting common patterns. If 'groebner', apply
-    an experimental groebner basis algorithm. In this case further options
-    are forwarded to ``trigsimp_groebner``, please refer to its docstring.
-    If 'combined', first run the groebner basis algorithm with small
-    default parameters, then run the 'matching' algorithm. 'fu' runs the
-    collection of trigonometric transformations described by Fu, et al.
-    (see the `fu` docstring).
-
+    """Returns a reduced expression by using known trig identities.
+
+    Parameters
+    ==========
+
+    method : string, optional
+        Specifies the method to use. Valid choices are:
+
+        - ``'matching'``, default
+        - ``'groebner'``
+        - ``'combined'``
+        - ``'fu'``
+        - ``'old'``
+
+        If ``'matching'``, simplify the expression recursively by targeting
+        common patterns. If ``'groebner'``, apply an experimental groebner
+        basis algorithm. In this case further options are forwarded to
+        ``trigsimp_groebner``, please refer to its docstring. If
+        ``'combined'``, it first runs the groebner basis algorithm with small
+        default parameters, then runs the ``'matching'`` algorithm. If ``'fu'``
+        runs the collection of trigonometric transformations described by Fu,
+        et al. (see the ``fu`` docstring). If ``'old'`` the original SymPy trig
```

Comment:
Is it possible to link to the fu method? Other than that LGTM!

Reply:
done